stdlib/stringLabels.mli

```diff
@@ -151,7 +151,7 @@ val compare : t -> t -> int
 
 val starts_with :
   prefix (* comment thwarts tools/sync_stdlib_docs *) :string -> string -> bool
-(** [starts_with ][~][prefix s] is [true] if and only if [s] starts with
+(** [starts_with ~prefix s] is [true] if and only if [s] starts with
```

Comment:
This line was correct - the same pattern is needed with `ends_with` below (and then _the same_ text appears in `string.mli`)